frontend/src/components/Footer.svelte

```diff
@@ -0,0 +1,114 @@
+<script>
+  import { createEventDispatcher } from 'svelte/internal';
+  import Button from './Button';
+
+  const dispatch = createEventDispatcher();
+
+  function clickHandler() {
+    dispatch('editName', true);
+  }
+</script>
+
+<footer>
+  <div>
+    <p>Room Availability:</p>
+    <div class="colors">
+      <p class="color-splash available">free</p>
+      <p class="color-splash taken">taken</p>
+      <p class="color-splash full">full</p>
+    </div>
+  </div>
+  <div>
+    <Button on:click={clickHandler} title="Edit Name" />
```

Comment:
It would be nice if we also could show the current name in the footer (next to the button)